### PR TITLE
Run upstream analysis

### LINKED ISSUE #70: Run the `upstream-analysis` module in the container
## What are the scientific goals of the analysis?

To run the `upstream-analysis` module in the container. We want to verify the container runs smoothly for all the steps/R packages for the `upstream-analysis` module. Please notice that we will run all steps in the module, but for the final object and report, we will not consider results from `SoupX` and `scDblFinder` for filtering purposes.

### PR DESCRIPTION
**Pull request by AntoniaChroni**
# Purpose/implementation Section

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## What scientific question/module of analysis is your analysis addressing?

To run the `upstream-analysis` module in the container. We want to verify the container runs smoothly for all the steps/R packages for the `upstream-analysis` module. Please notice that we will run all steps in the module, but for the final object and report, we will not consider results from `SoupX` and `scDblFinder` for filtering purposes.

## What GitHub issue does your pull request address?

Closes #70.

## Directions for reviewers 
Tell potential reviewers what kind of feedback you are soliciting.

### Which areas should receive a particularly close look?

Please submit the `lsf-script.txt` following the instructions in the `README` file and confirm that results, plots, and reports are generated for each step of the module pipeline. 

## Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

## Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by AntoniaChroni** [comment]
@sharonfreshour Please note minor edits on the `README` files about the container. See 19606741f229af884c19b85b7fb48a4a05259e98. Please provide feedback on the clarity.

File: project_parameters.Config.yaml [hunk 1]
```diff
@@ -51,6 +51,6 @@ use_condition_split_filter_object: "YES" # set for 04_run_filter_object.Rmd
 print_pdf_filter_object: "NO" # set for 04_run_filter_object.Rmd
 use_scDblFinder_filtering_filter_object: "NO" # set for 04_run_filter_object.Rmd
```
Comment (sharonfreshour) [inline project_parameters.Config.yaml:52]: The use_scDblFinder_filtering_filter_object param is duplicated in the yaml. One of the lines needs to be removed for lsf script to run.

File: analyses/fastqc-analysis/README.md
Comment (sharonfreshour) [inline analyses/fastqc-analysis/README.md:1]: Updates look good and improve clarity for both READMEs

File: project_parameters.Config.yaml (hunk 1 above)
Comment (AntoniaChroni) [reply project_parameters.Config.yaml:52]: Fixed here 0149196e6430781d9c778bcc9f352b65351ec9ad.

**Comment by sharonfreshour** [comment]
@AntoniaChroni the module appears to have run successfully through the `SoupX` step but ran into an error at the `scDblFinder` step

This is the error message given in job.err file:
```
processing file: 03_run_scDblFinder.Rmd
Error in `gzfile()`:
! invalid 'description' argument
Backtrace:
  1. rmarkdown::render(...)
  2. rmarkdown::render(...)
  3. knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet)
  4. knitr:::process_file(text, output)
  7. knitr:::process_group(group)
     ...
 15. base::withRestarts(...)
 16. base (local) withRestartList(expr, restarts)
 17. base (local) withOneRestart(withRestartList(expr, restarts[-nr]), restarts[[nr]])
 18. base (local) docall(restart$handler, restartArgs)
 20. evaluate (local) fun(base::quote(`<smplErrr>`))

Quitting from lines 202-310 [run-recoverDoublets-plot-predictions] (03_run_scDblFinder.Rmd)
Execution halted
```

Let me know if there is additional information I need to give you.

**Comment by AntoniaChroni** [comment]
> @AntoniaChroni the module appears to have run successfully through the `SoupX` step but ran into an error at the `scDblFinder` step
> 
> This is the error message given in job.err file:
> 
> ```
> processing file: 03_run_scDblFinder.Rmd
> Error in `gzfile()`:
> ! invalid 'description' argument
> Backtrace:
>   1. rmarkdown::render(...)
>   2. rmarkdown::render(...)
>   3. knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet)
>   4. knitr:::process_file(text, output)
>   7. knitr:::process_group(group)
>      ...
>  15. base::withRestarts(...)
>  16. base (local) withRestartList(expr, restarts)
>  17. base (local) withOneRestart(withRestartList(expr, restarts[-nr]), restarts[[nr]])
>  18. base (local) docall(restart$handler, restartArgs)
>  20. evaluate (local) fun(base::quote(`<smplErrr>`))
> 
> Quitting from lines 202-310 [run-recoverDoublets-plot-predictions] (03_run_scDblFinder.Rmd)
> Execution halted
> ```
> 
> Based on some brief research of this error message, it seems like it might be an error related to an incorrect file type?
> 
> Let me know if there is additional information I need to give you.

There was a typo in the filename. See 99c80a5b4e31e1b937e31dffae583c8adad33a71.

**Review by sharonfreshour — APPROVED** [review]
Per discussion

**Comment by AntoniaChroni** [comment]
@sharonfreshour The value for  [future.globals.maxSize](https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap/pull/71/commits/40cba40e1961b9c897eead486996866e1a10f218) is updated. We will close this for now and troubleshoot more if needed later.

**Pull request merged**